packages/tokens/src/global/breakpoints.ts

```diff
@@ -0,0 +1,101 @@
+export interface BreakpointsProps {
+  xxs: {
+    width: number;
+    margin: number;
+    gutter: number;
+  };
+
+  xs: {
+      width: number;
+      margin: number;
+      gutter: number;
+  };
+
+  sm: {
+      width: number;
+      margin: number;
+      gutter: number;
+  };
+
+  md: {
+      width: number;
+      margin: number;
+      gutter: number;
+  };
+
+  lg: {
+    width: number;
+    margin: number;
+    gutter: number;
+  };
+
+  xl: {
+    width: number;
+    margin: number;
+    gutter: number;
+  };
+
+  xxl: {
+    width: number;
+    margin: number;
+    gutter: number;
+  };
+
+  xxxl: {
+    width: number;
+    margin: number;
+    gutter: number;
+  };
+}
```

Comment:
```suggestion
type BreakpointType = 'xxs' | 'xs' | 'sm' | 'md' | 'lg' | 'xl' | 'xxl' | 'xxxl';

export interface Breakpoint {
  width: number;
  margin: number;
  gutter: number;
}

export interface BreakpointsProps {
  [key: BreakpointType]: Breakpoint;
}
```

Reply:
Great suggestion!

Reply:
Maybe we don't need to `export interface Breakpoint`